test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -265,31 +265,28 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			util.ExpectAdmittedWorkloadsTotalMetric(prodClusterQ, 2)
 			time.Sleep(podsReadyTimeout)
 			util.FinishEvictionForWorkloads(ctx, k8sClient, prodWl)
+
 			util.ExpectWorkloadsToHaveQuotaReservation(ctx, k8sClient, prodClusterQ.Name, prodWl)
 			util.ExpectQuotaReservedWorkloadsTotalMetric(prodClusterQ, 3)
 			util.ExpectAdmittedWorkloadsTotalMetric(prodClusterQ, 3)
-			time.Sleep(podsReadyTimeout)
 			ginkgo.By("evicted re-admitted workload should have 2 in the re-queue count")
 			util.ExpectWorkloadToHaveRequeueCount(ctx, k8sClient, client.ObjectKeyFromObject(prodWl), ptr.To[int32](2))
+			time.Sleep(podsReadyTimeout)
```

Comment:
is this a real sleep?
can it be made a few miliseconds just for this test?

Reply:
ping @tenzen-y

Reply:
@alculquicondor Thank you for reminding this. Does it mean that decreasing the timeout would be better?

https://github.com/kubernetes-sigs/kueue/blob/bd83e79ad4b9ac881004cefd35331f1b7a1f5a73/test/integration/scheduler/podsready/scheduler_test.go#L196

Reply:
yes. 3s is too long

Reply:
Let try it.

Reply:
yeah, I'm even wondering if we could reduce 3s for other places, maybe we should review that